android/build.gradle

```diff
@@ -1,7 +1,6 @@
-def DEFAULT_COMPILE_SDK_VERSION             = 23
-def DEFAULT_BUILD_TOOLS_VERSION             = "23.0.3"
-def DEFAULT_TARGET_SDK_VERSION              = 22
-def DEFAULT_GOOGLE_PLAY_SERVICES_VERSION    = "+"
+def safeExtGet(prop, fallback) {
+    rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
```

Comment:
very interesting approach, there were some complains around this, asking us to downgrade maybe this would be the solution?

We gotta test this to make sure it doesn't break existent examples. Thank you for the PR. We'll try to test this over the weekend.